src/base.iced

```diff
@@ -197,17 +197,40 @@ class Base
       new Error "Wrong seq_type: wanted '#{a}' but got '#{b}'"
     else if not (key = json?.body?.key)?
       new Error "no 'body.key' block in signature"
+    else if stanza_error = @_check_stanzas(json)
+      stanza_error
     else
       err = @_v_check_key key
     cb err
 
   #------
 
+  _required_stanzas : () -> ["key", "type", "version"]
+
+  #------
+
+  _optional_stanzas : () -> ["client", "merkle_root"]
```

Comment:
we'd have to change: (1) the web site; (2) old versions of node client; and (3) the "hardcore" mode of the Web site.